web/website/content/_index.md

```diff
@@ -169,8 +169,8 @@ showcase_section:
         SELECT
           *,
           started + unfinished AS finished,
-          (started + unfinished) / started AS fin_share,
-          (started + unfinished) / started / (1 - (started + unfinished) / started) AS fin_ratio
+          (started + unfinished * 1.0 / started) AS fin_share,
+          ( (started + unfinished * 1.0 / started) * 1.0 / 1 - (started + unfinished * 1.0 / started) ) AS fin_ratio
```

Comment:
Re the decision of when we do this coercion — overall I think this looks much worse to a casual new potential user — it's not immediately obvious to them why all these `* 1.0`, even if it does produce more consistent results. (I still think the int division is great tbc, always a tradeoff between precision and easy SQL...!)